install.sh

```diff
@@ -288,30 +333,32 @@ fi
 
 # Move the cloned git repository to the catkin workspace in semaphore
 if (env | grep SEMAPHORE | grep --quiet -oe '[^=]*$'); then
-	if [ -d ~/Navigator ]; then
-		mv ~/Navigator $CATKIN_DIR/src
-	elif [ -d ~/Sub8 ]; then
+	if [ -d ~/Sub8 ]; then
 		mv ~/Sub8 $CATKIN_DIR/src
+	elif [ -d ~/Navigator ]; then
+		mv ~/Navigator $CATKIN_DIR/src
 	fi
 fi
 
 # Source the workspace's configurations for bash on this user account
 source $CATKIN_DIR/devel/setup.bash
 if !(cat $BASHRC_FILE | grep --quiet "source $CATKIN_DIR/devel/setup.bash"); then
+	echo "" >> $BASHRC_FILE
+	echo "# Sets up the shell environment for the $CATKIN_DIR workspace" >> $BASHRC_FILE
 	echo "source $CATKIN_DIR/devel/setup.bash" >> $BASHRC_FILE
 fi
 
-# Check if the Navigator repository is present; if it isn't, download it
-if ($INSTALL_NAV) && !(ls $CATKIN_DIR/src | grep --quiet "Navigator"); then
-	instlog "Downloading the Navigator repository"
+# Download the software-common repository if it has not already been downloaded
+if !(ls $CATKIN_DIR/src | grep --quiet "software-common"); then
+	instlog "Downloading the software-common repository"
 	cd $CATKIN_DIR/src
-	git clone -q https://github.com/uf-mil/Navigator.git
-	cd $CATKIN_DIR/src/Navigator
+	git clone -q https://github.com/uf-mil/software-common.git
```

Comment:
Whenever there is an update to a branch, Semaphore drops the repository checked out to that branch in the user's home directory. We need to move it to $CATKIN_DIR/src for the build to work.